students/TylerRay/session3/listslab.py

```diff
@@ -0,0 +1,69 @@
+strChoice = True
+list = ['apples', 'pears', 'oranges', 'peaches']
+while strChoice != False:
+
+    print("""
+    Menu of Options
+    1) Show current list of fruit
+
+    2) Add a fruit to the end of the list.
+
+    3) Find fruit by number
+
+    4) Add fruit to the beginning of the list
+
+    5) Delete the first fruit
+
+    6) Delete fruit by number
+
+    7) Exit Program
+    """)
+    strChoice = str(input("Which option would you like to perform? [1 to 4] - "))
```

Comment:
You are using "Hungarian Notation: here. This is generally not recommend in Python. You can read ahead a bit about naming things here:

https://uwpce-pythoncert.github.io/PythonCertDevel/modules/NamingThings.html